core/trino-main/src/main/java/io/trino/execution/CreateViewTask.java

```diff
@@ -74,9 +87,9 @@ public ListenableFuture<Void> execute(
             List<Expression> parameters,
             WarningCollector warningCollector)
     {
-        if (!statement.getProperties().isEmpty()) {
-            throw semanticException(NOT_SUPPORTED, statement, "Creating views with properties is not supported");
-        }
+        Metadata metadata = plannerContext.getMetadata();
+        Map<NodeRef<Parameter>, Expression> parameterLookup = bindParameters(statement, parameters);
```

Comment:
the bindParameters function seems to be called twice within this method (line 91 , line 112)with same arguments, is it feasible to get rid the second call and reuse result from the first call?